config/tarball/exclude

```diff
@@ -1,20 +1,13 @@
 *.DS_Store
-*/examples
-*/local-extractor
-*/rdoc
 */spec
 */test
+*/tmp/*
 */VMwareWebService/**/*.wsdl
 *dos_mbr.img
-*Gemfile.appliance_excludes.rb
 *test.rb
-.git
-*system/initialize_appliance.sh
-*vmdb/Gemfile.lock
-*vmdb/.bundle
-*vmdb/bin
-*vmdb/config/database.yml
-*vmdb/certs/v2_key
-*vmdb/log/*.log
-*vmdb/tmp/miq_temp
-*lib/disk/modules/ruby1.9.3
+*.git
+Gemfile.lock*
+.bundle
+bin
```

Comment:
We were just discussing this...for right now we are going to exclude it because the gemset builder will put it right back in with a symlink.  We have to change the gemset builder to handle this differently, but for now we are excluding it.